docs/guides/ingesters-with-wal.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Ingesters with WAL"
+linkTitle: "Ingesters with WAL"
+weight: 5
+slug: ingesters-with-wal
+---
+
+Currently the ingesters without the TSDB stores all the data in the memory and if it happens to crash for any reason, there will be a loss of data. WAL helps fill this gap of reliability.
```

Comment:
Currently the ingesters running in the chunks storage mode, store all their data in memory. If there is a crash, there could be loss of data. ... gap _in_ reliability.